site/layouts/partials/scripts.html

```diff
@@ -18,3 +18,39 @@
 {{- end }}
 
 <script src="{{ $docsJs.Permalink | relURL }}"></script>
+
+
+{{ printf .Site.Params.cdn.css | safeHTMLAttr }}
+
+<script src="https://cdn.jsdelivr.net/npm/@stackblitz/sdk@1.5.3/bundles/sdk.umd.min.js"></script>
```

Comment:
We don't need to load this in every page. Which probably means, you shouldn't add the logic in application.js.

Reply:
Which file do you suggest I add this logic?

Reply:
Would adding a script tag to the `example.html` partial and putting "open in stackblitz" logic there be fine?